tests/integration/flows/resource-demand.yml

```
@@ -1,6 +1,7 @@
 jtype: Flow
 executors:
-  - name: cliptextencoder
-    uses: jinahub+docker://CLIPTextEncoder
+  - name: executor1
+    uses: jinahub+sandbox://Sentencizer
```

```suggestion
    uses: jinahub+docker://Sentencizer
```
`sandbox` will run in Hubble-provided sandbox & won't use the desired cpu.